doc/rst/api_xua_conf.rst

```diff
@@ -119,4 +119,5 @@ Power
 -----
 
 .. doxygendefine:: XUA_POWERMODE
+.. doxygendefine:: CHAN_BUFF_CTRL
```

Comment:
XUA_LOW_POWER_NON_STREAMING also. Again, I thought I had done this - apologies. Good catch

Reply:
Done